include/dlaf/factorization/qr/t_factor_impl.h

```diff
@@ -10,34 +10,33 @@
 
 #pragma once
 
-#include <pika/future.hpp>
-
 #include <blas.hh>
 
+#include <pika/future.hpp>
+
 #ifdef DLAF_WITH_GPU
 #include <whip.hpp>
 #endif
```

Comment:
If I'm not wrong, this should be moved at the very end in the "conditionally included" section.